extra-misc/hollywood/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=hollywood
+PKGSEC=extra
```

Comment:
`extra` is not a valid section. Please refer to `sets/section` for a list of valid sections. This probably goes into `utils`.

Reply:
I'm not sure where this should go & extra seems to be a good target. WIP

Reply:
Extra is not even a valid section. Try `utils`.